compiler-core/src/language_server/code_action.rs

```diff
@@ -813,6 +969,12 @@ pub fn code_action_add_missing_patterns(
             indent_size += 1;
         }
 
+        // TODO: It would be nice to use the `print_case_expression` function here,
+        // however that causes things like comments to be removed, which is undesirable.
+        // In future we should work out a way to use that function, as the code below is
+        // pretty convoluted and difficult to understand, because it accounts for a set of
+        // very specific edge-cases. A generic formatting function would be much better.
```

Comment:
Don't add open ended todos comments please

Reply:
Alright. Just a thought I had. Should I leave the comment there without the `TODO`, or remove it entirely?

Reply:
Remove it entirely, it's not actionable and work to go should go in the issue tracker